cypress/README.md

```diff
@@ -0,0 +1,123 @@
+## E2E tests
```

Comment:
thank you for so thoroughly updating this!

Reply:
:sweat_smile:

Reply:
+1, this is awesome! Once this is merged, would you mind also adding it to the wiki? (or just linking to this README somewhere on our getting started wiki page)